issues-policy.md

```diff
@@ -0,0 +1,84 @@
+# GitHub issues process and policy
+
+The goals of the process are:
+
+1. Ensure errors or omissions in our docs are not blocking customer success.
+1. Be responsive to customer feedback and concerns.
+1. Continually improve the customer experience.
+1. Learn more about customer experiences through dialogue in the open on challenges and solutions.
+
+The process uses two stages to ensure responsiveness while prioritizing work. The initial stage diagnoses and triages the issue. The second stage resolves the issue. When an issue is both easy and urgent, the two stages may be combined.
+
+The process involves two time boxed tasks:
+
+- Each team member spends up to 1/2 hour each business day [classifying incoming issues](#diagnosis-phase), including initial response. This ensures we are responsive on new issues.
+- Each team member spends up to 1.5 days per sprint [updating documents](#resolution-phase) fixing customer generated GitHub issues.
+
+## Diagnosis phase
+
+Every team member spends up to 30 minutes per business day categorizing new issues. We answer the following questions:
+
+- Is the issue a docs issue, or a product issue?
+- Is it not an issue, but a question better suited for a forum or support site?
+- What priority is the issue?
+- What area manages this issue?
+
+If these questions can't be answered in the initial examination, we ask clarifying questions in the comments.
+
+There are types of issues that are closed during the diagnosis and triage phase:
+
+- **kudos**: Express thanks, and close the issue.
+- **product issue**: Issues that request product changes are closed. We may also take additional actions, as described below.
+- **CoC violations**: These issues are closed, and reported if the [CoC violation](https://dotnetfoundation.org/code-of-conduct) merits reporting and blocking.
+- **duplicates**: Duplicates are closed with a comment referencing the existing issue.
+- **doc-ok**: The customer is incorrect, and the doc is correct.
+- **forum**: Issues that are support or forum requests are directed to Stack Overflow, or other support sites, and closed.
+
+### Actions on product issues
+
+Depending on the nature of the product issue, we may choose to:
+
+- Transfer the issue to the appropriate product repository.
+- Close the issue as a duplicate of existing product requests.
+- Close the issue with a recommendation to open on the product repository.
+
+Evaluating the correct course of action is subjective. The team members use their judgment on the correct action.
+
+### Actions on content issues
+
+For other issues, the team:
+
+- assigns a priority
+- assigns a milestone, usually "Backlog"
+- reflects tier level support
```

Comment:
reflects --> selects or assigns?
"tier level support" is an award phrase. Is this priority level?

Reply:
I deleted that bullet item. It's part of an earlier discussion that no longer applies.